lib/MusicBrainz/Server/Data/Recording.pm

```diff
@@ -422,9 +422,8 @@ sub find_recent_by_artists
 {
     my ($self, $artist_ids) = @_;
 
-    my $search_url = sprintf("http://%s/ws/2/recording/?query=title:\"\"",
-                              DBDefs->LUCENE_SERVER);
-
+    my $search_url = sprintf("http://%s/recording/select?q=recording:*&wt=mbxml",
+                                 DBDefs->SOLR_SERVER,);
```

Comment:
Not that perl cares, but you added a trailing comma there which looks weird.

Reply:
old habit from python dicts. easier to add to it. Will fix.

Reply:
It would definitely make sense if `);` were on its own line, since then additions or removals at the end would only be a one-line change instead of two. Sorry, I was just being picky.